velox/vector/BaseVector.cpp

```diff
@@ -437,6 +437,28 @@ std::string BaseVector::toString() const {
     out << countNulls(nulls_, 0, length_) << " nulls";
   }
   out << "]";
+
+  return out.str();
+}
+
+std::string BaseVector::toString(bool recursive) const {
```

Comment:
I believe these are changes from #2206 .

Reply:
This PR depends on 2 other PRs: #2206 and #2205. The last commit contains the actual changes.

Reply:
Thanks Masha, will wait for those 2 prs to be merged.